src/types/Theme.ts

```diff
@@ -1,6 +1,8 @@
 import { asNumber, asObject } from 'cleaners'
 import { ImageSourcePropType } from 'react-native'
```

Comment:
Weren't you going to remove `ImageSourcePropType`, and just use the `ImageProp`?